examples/metricq_sink.py

```diff
@@ -76,11 +69,8 @@ async def on_data(
         )
 
 
-@click.command()
-@click.option("--server", default="amqp://localhost/")
-@click.option("--token", default="sink-py-dummy")
+@metricq_command(default_token="sink-py-dummy")
 @click.option("-m", "--metrics", multiple=True, required=True)
```

Comment:
I'd like to see a `@metricq_metrics_options`, or even better `@metricq_metric_option(multiple=True)`